common/ledger/blkstorage/blockstore_provider_test.go

```diff
@@ -191,6 +191,17 @@ func TestBlockStoreProvider(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, false, exists)
 
+	// remove a channel and verify blockindex and blocks dir are deleted
+	ledgerid := constructLedgerid(0)
+	provider.Remove(ledgerid)
+	iter := provider.leveldbProvider.GetDBHandle(ledgerid).GetIterator(nil, nil)
+	require.False(t, iter.Next())
```

Comment:
Isn't the index store already empty for the ledgerID as there is no block/tx?

Reply:
The purpose is to verify that no entries are found for the ledgerid in blockindex leveldb.